packages/core/elasticsearch/core-elasticsearch-client-server-internal/src/client_config.ts

```diff
@@ -24,13 +24,15 @@ export type ParsedClientOptions = Omit<ClientOptions, 'agent'> & { agent: HttpAg
  */
 export function parseClientOptions(
   config: ElasticsearchClientConfig,
-  scoped: boolean
+  scoped: boolean,
+  kibanaVersion: string
 ): ParsedClientOptions {
   const clientOptions: ParsedClientOptions = {
     sniffOnStart: config.sniffOnStart,
     sniffOnConnectionFault: config.sniffOnConnectionFault,
     headers: {
       ...DEFAULT_HEADERS,
+      ...{ [USER_AGENT_HEADER]: `Kibana/${kibanaVersion}` },
```

Comment:
So in the end we're only sending `Kibana/{KIB_VERSION}` and not `Kibana/{KIB_VERSION}; Elasticsearchjs/{CLIENT_VERSION}` ? (I'm fine with it, just asking)